### PR TITLE
NEWS.md: add release notes for v0.18.0

### PR DESCRIPTION
**Pull request by jameshcorbett**
Waiting on #237 to go in.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+flux-coral2 0.18.0 - 2024-11-07
```
Comment (cmoussa1) [inline NEWS.md:1]: you can probably change this date to today's, 2024-11-08

**Review by cmoussa1 — APPROVED** [review]
LGTM 🎉

File: NEWS.md (hunk 1 above)
Comment (jameshcorbett) [reply NEWS.md:1]: Oh duh thanks

Comment (jameshcorbett) [reply NEWS.md:1]: will fix and set MWP

**Pull request merged**